server/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Jim-Call-Automation",
```

Comment:
Actually this is not related to Jim. You can call it **Call-Automation** tout court.